src/Database/Builder.php

```diff
@@ -14,6 +14,8 @@
  */
 class Builder extends BuilderModel
 {
+    use Eloquent\Concerns\QueriesRelationships;
```

Comment:
@Klaasie while this is technically correct, our current convention is to use the fully namespaced trait name, just to make it clear whether it's an October or Laravel class.

Reply:
I've changed the statement :)